### PR TITLE
Added support to compile on Windows + MSYS2 (WIP)

### PR DESCRIPTION
**Pull request by CicTec**

### CODE REVIEW

File: src/divforma.c [hunk 1]
```diff
@@ -37,15 +37,15 @@ static jmp_buf jmp_error_ptr;
 // ����������������������������������������������������������������������������
 //      Estructuras
 // ����������������������������������������������������������������������������
-
+#if !defined(_WIN32) && !defined(WIN32)
```
Comment (MikeDX) [inline src/divforma.c]: in src/divforma.c you have commented out code for tagRGBQUAD and tagBITMAPFILEHEADER. Do we need this or are you using another bitmap struct somewhere else in the msys version?

Comment (CicTec) [reply src/divforma.c]: I have commented because cause compile error: struct already declared.

**Comment by CicTec** [comment]
Ok, i have modified BITMAP and RGBQUAD struct because are defined in wingdi.h that is included automatically on windows build, now there is no longer any build conflict.

**Comment by MikeDX** [comment]
I've had to revert part of this commit as it broke the nightly cross compiled builds. The bitmap struct changes have been left intact

**Pull request merged**